ablog/__init__.py

```diff
@@ -62,6 +72,52 @@ def html_page_context(app, pagename, templatename, context, doctree):
             context["feed_title"] = blog.blog_title
 
 
+def builder_inited(app):
+    if not app.config.inject_templates_after_theme:
```

Comment:
You'll want to register this variable with add_config or whatever that method is called, on the `app` as part of the setup. +

Reply:
This should now be the case.